lib/src/kvs/rocksdb/mod.rs

```diff
@@ -286,3 +291,18 @@ impl Transaction {
 		Ok(res)
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	// https://github.com/surrealdb/surrealdb/issues/76
+	#[tokio::test]
+	async fn soundness() {
+		let mut transaction = get_transaction().await;
+		transaction.put("uh", "oh").await.unwrap();
+
+		async fn get_transaction() -> crate::Transaction {
+			let datastore = crate::Datastore::new("rocksdb:/tmp/rocks.db").await.unwrap();
```

Comment:
The path used here may be a compatibility concern, in which case the entire test can be commented out.

Reply:
Hi @finnbear, yeah just thinking about this part. Can you think of any reason why this might not work in CI (mac/linux/windows)?

Reply:
`/tmp` exists on all Linuxes I am aware of, but not necessarily on windows or mac (don't have equipment to test on). I could add a dev dependency to portable get a temporary path, or comment out/remove the test.

Reply:
Let's leave it as you've done it and I'll test the CI after it's merged!